setup.py

```diff
@@ -99,6 +115,48 @@ def run(self) -> None:
         cmake_cmd = ["cmake", "--build", "."] + self.build_args
         subprocess.run(cmake_cmd, cwd=self.build_temp, check=True)
 
+        if editable_mode:
+            # NOTE(brad) When installed in editable mode, python expects to find symengine
+            # in the symenginepy/symengine source directory. Everything is already there
+            # except the compiled symengine_wrapper extension module, so we copy that there
+            # as well.
+            # NOTE(brad) For some reason that I don't fully understand, the symengine_wrapper
+            # shared library is neither present in symengine_install nor needed in the source
+            # directory on macos sometimes (for example, on the github actions macos runner).
+            # So, if the copy fails (due to the file not being present, we just move on).
+            try:
+                self.copy_file(
+                    build_temp_path
+                    / "symengine_install"
+                    / "lib"
+                    / f"python{sys.version_info.major}.{sys.version_info.minor}"
+                    / "site-packages"
+                    / "symengine"
+                    / "lib"
+                    / self.get_ext_filename("symengine_wrapper"),
+                    SOURCE_DIR
+                    / "third_party"
+                    / "symenginepy"
+                    / "symengine"
+                    / "lib"
+                    / self.get_ext_filename("symengine_wrapper"),
+                )
+            except:
+                pass
+
+            # NOTE(brad) By setting the RPATH of the generated binaries to include $ORIGIN,
+            # we told all binaries they can expect to find any shared libraries in the same
+            # directory they are already in. In particular, we told cc_sym.so it can find
+            # its dependencies (the below shared libraries) in SOURCE_DIR (where we put it).
+            # Thus, we need to copy the shared libaries there as well so that cc_sym.so will
+            # find them when it looks.
+            shared_library_extension = ".dylib" if sys.platform == "darwin" else ".so"
+            for cc_sym_dependency in ["libsymforce_gen", "libsymforce_opt", "libsymforce_cholesky"]:
```

Comment:
Can we glob `build_temp_path` for things that match `"lib*{shared_library_extension}"`, or `"libsymforce*{shared_library_extension}"`?  I'm worried about keeping this up to date

Reply:
Good idea.